interview-questions.md

```diff
@@ -6,41 +6,54 @@ First, without external resources. Challenge yourself to answer from memory as i
 
 Then, research the question to expand on your answer. Even if you feel you have answered the question completely, there is always something more to learn. Write your researched answer in your OWN WORDS.
 
+
+
+
+
+
 1. What is the difference between a parameter and an argument?
 
 Your answer:
+the best way i can explain an argument is using an arugument is  {const example = (parameter )=>{
+return( argument 1, argument 2 )
+}}
 
 Researched answer:
+The values that are declared within a function when the function is called are known as an argument. Whereas, the variables that are defined when the function is declared are known as a parameter.
+
+
 
 2. The JavaScript built in method .map() takes predefined parameters. What are they? Which are required and which are optional?
 
-Your answer:
+Your answer: index, value and array. The required parameter is  the value.
+
+Researched answer:  The map() method is a copying method. It does not alter (this)
```

Comment:
This is a perfect place to expand on what you already know. How are index and array useful if used as parameters in the map method?